Kernel/Graphics/FramebufferDevice.cpp

```diff
@@ -99,14 +99,21 @@ UNMAP_AFTER_INIT KResult FramebufferDevice::initialize()
     return KSuccess;
 }
 
+UNMAP_AFTER_INIT FramebufferDevice::FramebufferDevice(const GraphicsDevice& adapter, size_t output_port_index)
+    : BlockDevice(29, GraphicsManagement::the().allocate_minor_device_number())
```

Comment:
We might want to make this 29 a constant.

Reply:
I'm not sure this is the right thing to do, especially after the last push of commits (where I simplified the ioctl interface and created a "pure virtual" `GenericFramebufferDevice`). It's not like we really rely on this number in any other place (as far as I remember), and this number is also somewhat arbitrary.

Reply:
I think it might be a good idea to eventually make an enum somewhere where we give names to the major and minor numbers we allocate between devices. This may be a large change effecting multiple subsystems though, so maybe it should be done in a different PR.

Reply:
I agree, another "item for the yak stack" as people like to say here :)